client/app/scripts/superdesk-authoring/authoring.js

```diff
@@ -1600,6 +1670,19 @@
                             scope.origItem.dateline.source, scope.origItem.dateline.date);
                     }
                 };
+
+                scope.evalAspectRatio = function(ar) {
+                    ar = ar.replace('-', '/');
+                    var Fn = Function, val = new Fn('return ' + ar)();
```

Comment:
What happens if `ar` ends up being something like `'3/4 && console.log("Boo")';`, for any reason? It's too dangerous to have a risk like that, effectively calling `eval()` here.

I suggest to simply extract the values of `width` and `height` and perform the division explicitly, e.g.:
```js
var parts = ar.split('-').map(_.partial(parseInt, _, 10));
// TODO: add some checks for errors, zero division etc...
return parts[0] / parts[1];
```